configs/projects/pe-bolt-server-runtime-main.rb

```diff
@@ -1,6 +1,6 @@
 project 'pe-bolt-server-runtime-main' do |proj|
   proj.setting(:pe_version, 'main')
-  proj.setting(:rubygem_puppet_version, '7.12.0')
+  proj.setting(:rubygem_puppet_version, '7.13.1')
```

Comment:
This also needs to get updated in configs/projects/pe-bolt-server-runtime-2019.8.x.r

Reply:
Seems that was using the old version 6.x version, '6.25.0', okay to update to '7.13.1'?

Reply:
We absolutely do not want to ship puppet 7 in 2019.8.x!! This will break ace-server and will generally be bad.

Reply:
Reverted